setup.py

```diff
@@ -1,7 +1,24 @@
+import codecs
+import os
+
 import setuptools
 
-with open("README.md", "r") as fh:
-    long_description = fh.read()
+
+def read(rel_path):
```

Comment:
rel_path -> file path

Reply:
Thanks for the suggestion, it has been fixed in https://github.com/IBM/differential-privacy-library/pull/13/commits/ddb54916c91afa3a4513320edfb57daf1ebcf8e5